pom.xml

```diff
@@ -173,7 +174,7 @@
                 <groupId>org.projectlombok</groupId>
                 <artifactId>lombok</artifactId>
                 <version>1.18.24</version>
-                <scope>compile</scope>
+                <scope>provided</scope>
             </dependency>
```

Comment:
Changing to provided, since we don't need lombok jar during runtime.

Reply:
I think there may be a way to add this as a dependency for the step that does the codegen for lombok, can we try that?

Reply:
@the-other-tim-brown I couldn’t find a specific dependency scope in Maven that is used exclusively for the compile(lombok generates code during compilation) phase. In Gradle, the compileOnly scope serves this purpose, but in Maven, the closest equivalent is the provided scope, which works similarly to compileOnly by including dependencies during compile time without bundling them in the final artifact.

Reply:
As per the lombok docs, lombok maven dependency by default contains provided as the scope
https://projectlombok.org/setup/maven